contracts/ElectionPractice.sol

```diff
@@ -0,0 +1,166 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title - How to use this?
+ * @dev - deploy contract to DEXON netowkr
+ * @dev - copy deployed address to https://dexon-workshop-vote.netlify.com/
+*/
+
+/**
+ * @title - In this Election Contract, it could take as many rounds of election.
+ *        - As a user of this contract, you are able to
+ *        - 1. Register as a candiate of the election.
```

Comment:
should we mention about 'guarantee deposit ' here?

Reply:
no, we will move this part into different contract in order to expose floating point issue #3